src/Components/ArtworkAlert/Components/Steps/Confirmation.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Join, Separator, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+
+import { CriteriaPillsQueryRenderer } from "Components/ArtworkAlert/Components/CriteriaPills"
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+import { ConfirmationArtworksGridQueryRenderer } from "Components/SavedSearchAlert/ConfirmationArtworksGrid"
+
+export const Confirmation: FC = () => {
+  const { state } = useArtworkAlertContext()
```

Comment:
**walkthrough**: Again, similar to the component used in the current flow but here we use the new ArtworkAlertContext.